tests/foreman/ui/test_dashboard.py

```diff
@@ -14,13 +14,57 @@
 
 @Upstream: No
 """
+from robottelo.constants import ANY_CONTEXT
 from robottelo.decorators import stubbed, tier1, tier2
 from robottelo.test import UITestCase
+from robottelo.ui.factory import set_context
+from robottelo.ui.locators import common_locators
+from robottelo.ui.session import Session
 
 
 class DashboardTestCase(UITestCase):
     """Tests for Dashboard UI"""
 
+    @tier1
+    def test_positive_search_random(self):
```

Comment:
nope. Search should work whatever correct data I input. It would be negative if I input invalid or unexpected data